iotlabsshcli/open_a8.py

```diff
@@ -117,8 +119,55 @@ def wait_for_boot(config_ssh, nodes, max_wait=120, verbose=False):
     return {"wait-for-boot": result}
 
 
-def run_script(config_ssh, nodes, script, verbose=False):
-    """Run a script in background on the A8 nodes."""
+def run_cmd(config_ssh, nodes, cmd, frontend, verbose=False):
+    """ Run a command on the A8 nodes or on the
+    SSH frontend
+    """
+
+    # Configure ssh.
+    groups = _nodes_grouped(nodes)
+    ssh = OpenA8Ssh(config_ssh, groups, verbose=verbose)
+    try:
+        if frontend:
+            result = ssh.run(cmd, with_proxy=False)
+        else:
+            result = ssh.run(cmd)
+    except OpenA8SshAuthenticationException as exc:
+        print(exc.msg)
+        result = {"1": nodes}
+
+    return {"run-cmd": result}
+
+
+def copy_file(config_ssh, nodes, file_path, verbose=False):
+    """ Copy a file on the A8 SSH frontend directory
```

Comment:
on a8 ssh frontend_S_ (there might be several frontends involved in the experiment) ?